src/main/java/seedu/savenus/logic/parser/QuantifierParser.java

```diff
@@ -36,8 +24,7 @@
             + "Please fix the formatting to: FIELD QUANTIFIER VALUE\n"
             + "Note that VALUE MUST be only one word.";
     public static final String INVALID_FIELD_USAGE = "Note you have entered an invalid field! \n"
-            + "You are only allowed to enter the following fields:\n"
-            + "NAME, PRICE, CATEGORY, DESCRIPTION, LOCATION, OPENING_HOURS, RESTRICTIONS";
+            + "You are only allowed to enter PRICE as of v1.4";
```

Comment:
Just need to inform the user that "You can only filter food items based on price!" Do not need to include the v1.4 part as these should only be included in the User Guide to inform the users of upcoming changes.

Reply:
Thanks Clarence, will fix it in the next commit.